README.md

```diff
@@ -27,3 +27,48 @@ We want to provide a single input interface to the system (pipeline index) and d
     * Probably part of Input Interface, presenting logs from failed builds
 
 
+## Usage
+
+### Vagrant
+
+### Setup environment
+
+#### CentOS
+```
+# Install dependencies
+sudo yum install -y epel-release git
+sudo yum install -y ansible1.9 centos-release-scl qemu-kvm libvirt sclo-vagrant1
+
+# start libvirtd
+sudo systemctl start libvirtd; sudo systemctl enable libvirtd
+
+# enable bash on vagrant scl
+sudo scl enable sclo-vagrant1 bash
+```
+
+#### Fedora
+```
+# sudo dnf install -y git ansible vagrant
+```
+
+### Get the code and install vagrant plugins
+
+```
+git clone https://github.com/CentOS/container-pipeline-service
+cd container-pipeline-service
+vagrant plugin install vagrant-host-manager
```

Comment:
This fails for me : 
$ vagrant plugin install vagrant-host-manager
Installing the 'vagrant-host-manager' plugin. This can take a few minutes...
Bundler, the underlying system Vagrant uses to install plugins,
reported an error. The error is shown below. These errors are usually
caused by misconfigured plugin installations or transient network
issues. The error from Bundler is:

Could not find gem 'vagrant-host-manager (>= 0) ruby' in the gems available on this machine.